client/gutenberg/extensions/presets/jetpack/utils/register-jetpack-block.js

```diff
@@ -17,7 +17,7 @@ import isJetpackExtensionAvailable from './is-jetpack-extension-available';
  * @returns {object|false} Either false if the block is not available, or the results of `registerBlockType`
  */
 export default function registerJetpackBlock( name, settings, childBlocks = [] ) {
-	if ( ! isJetpackExtensionAvailable( name ) ) {
+	if ( ! getJetpackExtensionAvailability( name ).available ) {
```

Comment:
If we're going to use the unavailable reason later, it might be clearer to specify what this check means.
```jsx
const { available, unavailableReason } = getJetpackExtensionAvailability( name );
const unavailable = ! available;
```